.stylelintrc

```diff
@@ -1,9 +1,8 @@
 {
```

Comment:
I'm guessing this and all the other lint-related changes are due to an upgrade in eslint? It _appears_ we're dropping prettier for styles—is there a reason for that?

Reply:
> I'm guessing this and all the other lint-related changes are due to an upgrade in eslint?

Yes, and prettier and stylelint.

> It appears we're dropping prettier for styles—is there a reason for that?

Some lint in stylelint after upgrading complained about the way we were doing it being deprecated and causing conflicts and it appears the defaults are a lot better now, so I just removed it.

If you have the patience to figure out what's needed to bring back the previous setup, then go for it, but it didn't feel worth it to me after already spending 2x the amount of time on package breakage (mostly relating to linters and formatters) than I spent actually improving the code :)